File: C/practicals/Practical-2.c/p3.c
#include<stdio.h>
void main (){

	int x;
	scanf("%d",&x);

	if (x%7==0 && x>=21){
		printf("NUmber is divisible by 7 and greater than 21\n");
	}else if (x%7==0 && x<21){
		printf("NUmber is divisible by 7 and less than 21\n");
	}else{
		printf("Number is not divisible by 7\n");
	}
}	

File: JAVA/AsgnPract/3/p3.java
import java.util.*;

class IF{

	public static void main(String[] args){

		Scanner sc = new Scanner(System.in);
		
		System.out.print("Enter the number: ");

		int num = sc.nextInt();

		if (num > 10){

			System.out.println(num + " is greater than 10");
		
		}else if (num <10){
		
			System.out.println(num + " is less than 10");
		
		}else{
			System.out.println(num + " is equal to 10");
		}
	}
}

